test/background_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class BackgroundTest < Minitest::Test
+  include InvoicePrinterHelpers
+
+  def setup
+    @invoice = InvoicePrinter::Document.new(default_document_params)
+  end
+
+  def test_background_render
+    assert_silent do
```

Comment:
`assert_silent` is too strict and will fail when Prawn libraries output warnings. This test would be enough without this call e.g. remove `do...end` block just keep the calls to `#render` themselves.